concourse/client/util.py

```diff
@@ -67,6 +68,33 @@ def determine_jobs_to_be_triggered(
     )
 
 
+def job_has_been_triggered(
```

Comment:
this name is confusing, as it is merely a permutation of `has_job_been_triggered`

please try to choose expressive names that correlate with the implemented semantics

as I understand this function, it will _wait_, if required, for some time, until a given job of interest is triggered. so a better name should include this.

e.g. `wait_for_job_execution`, or `wait_for_job_to_be_triggered`, or sth. of the like

Reply:
done